### DISCUSSION TITLE
Do i need to learn laravel to use Livewire

### DISCUSSION THREAD

**Discussion by maheshkumar2150** [livewire/livewire#7443] [1. Help]
I haven't used php or laravel. I am using golang.

I am looking for a front end. I don't like the Javascript based fronted. They are difficult to implement server side auth. 

I found Livewire.
Do i need to learn laravel? Can i start right away?

Please guide..

**Comment by NBA707** [comment]
My vote would be to start right away.  Watch the videos available on livewire.laravel.com and see if you think it's worth the investment to learn.   No doubt it is in my mind, but your milage will vary.

Lots of You Tube vidoes on Laravel and Livewire to some degree.    

Should be enough to decide to deep dive and adopt as your stack

**Comment by joshhanley** [comment]
@maheshkumar2150 I would probably learn a bit of Laravel first, as Livewire is an extension of Laravel, so things may not make sense without some Laravel knowledge.

I would check out https://laracasts.com/series/laravel-8-from-scratch
It's for version 8 but there's not a lot of difference between 8 and 10, so going through that will give you the basics. And then I would look into Livewire.

Also check out the Livewire discord, people are generally pretty helpful if you get stuck https://discord.gg/livewire

**Comment by lrljoe** [comment]
For starters, if you've not used PHP before, I'd recommend you start with an "Intro To" approach, so that you learn the basics

Then as @joshhanley said, look at the Laracasts, v8 to v10 isn't massively different

Then you'll be ready to properly understand and implement Livewire without making the typical mistakes that you kick yourself for in 6 months time.

With Laravel/Livewire, always, **always**, look around at the vast number of open source packages, someone has almost always come across the issue before, and released a package that'll help smooth the way forward.

**Comment by NBA707** [comment]
Are you using golang as the backend or going to reduce yout stack to just php/laravel?  Livewire allows you to simply the stack by linking server side PHP to a client side web view.   Or is your server side PHP going to call goLang API?
